### ISSUE TITLE
The components of the 'STPPaymentCardTextField' should follow the contentVerticalAlignment of the parent

### ISSUE THREAD

**Issue by pblondin** [stripe/stripe-ios#217]
All my textfields need a bottom vertical aligment and when I set the 'STPPaymentCardTextField' to use UIControlContentVerticalAlignmentBottom the components do not follow.

For the moment, I have to do a dirty fix like this :
```
// align all components to bottom
for (UIView *subview in self.subviews) {
    if ([subview isKindOfClass:[UIControl class]]) {
        ((UIControl *)subview).contentVerticalAlignment = UIControlContentVerticalAlignmentBottom;
    }
}
```

and to handle the card brand image differently :
```
- (void)layoutSubviews {
    [super layoutSubviews];

    // we move the brandimage a little bit up
    self.brandImageView.frame = CGRectMake(10, -3, self.brandImageView.image.size.width, self.frame.size.height);
}
```

<img width="370" alt="example cc" src="https://cloud.githubusercontent.com/assets/1045929/9523682/432c8876-4ca9-11e5-8d89-9f0e4adcf02f.png">

I'll be happy to do a PR if you need help with this one! :)

**Comment by jflinter** [comment]
@pblondin, I've fixed this in 67f32a156546ac1e200e84707e7170e8ae891e17, let me know what you think!

**Comment by pblondin** [comment]
It fits perfectly my needs, again thanks a lot!
